データベース設計

nakamu26 opened the pull request:
# What
- ER図を作成
- READMEにテーブル設計を記載

# Why
DBにテーブルを作成するため

ER図のURL
https://gyazo.com/60944b51ceb43d904a1f70d017aa26d9

nakamu26 committed:
データベース設計

div-reviewer reviewed (commented):
nakamu26様、学習お疲れさまです。
DB設計の初回レビュー依頼頂きありがとうございました。プルリクエスト確認させていただきました。

テーブル間のアソシエーションまでよく考えられて作成されていますね！素晴らしいです。
修正点をコメントさせて頂きましたので、ご確認をお願い致します。

```ruby
【次回プルリクエスト提出する際の注意点】
①修正が終わりましたら、`コミットとプッシュ`をして、`プルリクエストに修正内容が反映されているか確認`しましょう。
②既に頂いているER図も修正範囲だった場合、次回プルリクエスト提出時には最新の画像の提出をお願い致します。
③googleフォームの送信をお願いいたします。
```

**ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください！**
どうぞよろしくおねがいします。

div-reviewer reviewed (commented):
■前回のレビューにこちらも追加でご確認をお願いします。

1件うまく貼り付けができていなかったため、再度送信させて頂きました。
お手数ですがあわせてのご確認をお願い致します。

nakamu26 committed:
データベース設計の修正１

- birth_dateカラムの型をdate型に修正。
- priceカラムの型をmediunint型に修正、unsignedオプションを追加。
- activehashを利用するカラムの名前を「xxx_id」に修正、型をinteger型に修正。

div-reviewer reviewed (commented):
nakamu26様、学習お疲れさまです。
DB設計の再レビュー依頼ありがとうございました。

前回記載させて頂いたコメントについてしっかり調べてreadmeやer図を修正されているのが伝わりました。素晴らしいです。
こちら`LGTM`とさせて頂きます。実装お疲れさまでした。

**●修正について●**
修正後は`再度レビュー依頼は出さなくて大丈夫です`ので、修正後、コミットとプッシュして頂けましたら、
マージ後、ローカルのmainブランチに忘れずにプルし、次の実装に取り組んで頂ければと思います。

ご不明点や疑問点ございましたら、チャット・コネクトメンターにご相談ください！
今後の実装も応援しております！どうぞよろしくおねがいします。

nakamu26 committed:
データベース設計修正２

- priceカラムの型をintegerに修正。
